lib/lev/status.rb

```diff
@@ -96,13 +145,13 @@ def has_reserved_keys?(hash)
     end
 
     def push(key, new_item)
-      new_value = (get(key) || []).push(new_item)
+      new_value = (send(key) || []).push(new_item)
       set(key => new_value)
     end
 
     STATES.each do |state|
       define_method("#{state}?") do
-        get('state') == state
+        status == state
```

Comment:
No I changed 'state' to 'status' because we also changed UUID to ID and because 'as_json' now returns the keys in the store with their values, it was simpler to just rename it rather than write in a conditional to re-write it in the 'as_json' method